src/types/index.ts

```diff
@@ -10,3 +10,6 @@ export enum VoteType {
   no = 'no',
   abstain = 'abstain',
 }
+
+export type Address = `0x${string}`;
```

Comment:
is the same as `import {Address} from viem` ?

Reply:
yeah oops I redefined my own before seeing wagmi/viem exposes the same type